chapters/improvements/sections/terrain_improvements.tex

```diff
@@ -0,0 +1,29 @@
+\section{Terrain improvements}
+The terrain in the game is quite interesting.
+It can be modified, it is procedurally generated and it is infinite.
+Moreover, there are five different biomes which adds some verity to the game.
+But of course there is always room for improvement.
+One such improvement would be to allow for multiple biomes in the same world.
+Right now the whole world is the same biome but it would be interesting to have different biomes blend into each other.
+
+Another improvement would be to add some structures to the world.
+Right now the world is just hills and valleys but it could include things like trees, rocks, buildings, roads, rivers, lakes, etc.
+In particular, lakes and rivers would be interesting as there is no water is the game as of now.
+Adding water and maybe even different fluids would be a great improvement.
+
+Terrain modification could also be improved.
+Right now the player cannot choose which materials to build with.
+It would allow for more interesting structures if the player could choose what type of material to place where.
+It would also make for more interesting gameplay if the player had to gather resources to build with instead of having an infinite supply of them.
+
+The last upgrade in the section would be to minimize the amount of space the terrain takes up.
```

Comment:
Maybe "The last upgrade concerning the terrain ... terrain takes up on disk when game saves are made."